org.jonata.Subtitld.yml

```
@@ -0,0 +1,487 @@
+app-id: org.jonata.Subtitld
+
+runtime: org.kde.Platform
+runtime-version: '6.3'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '6.3'
+
+command: subtitld
+rename-desktop-file: subtitld.desktop
+rename-icon: subtitld
+finish-args:
+  - --device=dri
+  - --filesystem=xdg-download
+  - --filesystem=xdg-music
+  - --filesystem=xdg-pictures
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
+
+modules:
+
+  - name: ffms2
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://github.com/FFMS/ffms2/archive/refs/tags/2.40.tar.gz
+        sha256: 82e95662946f3d6e1b529eadbd72bed196adfbc41368b2d50493efce6e716320
+
+  - name: libmpv
+    cleanup:
+      - "/include"
+      - "/lib/pkgconfig"
+    buildsystem: simple
+    build-commands:
+      - python3 waf configure --prefix=${FLATPAK_DEST} --enable-libmpv-shared --disable-cplayer --disable-alsa --disable-manpage-build --disable-debug-build
+      - python3 waf build
```

```suggestion
      - python3 waf -j ${FLATPAK_BUILDER_N_JOBS} build
```

This will make the build faster using parallel build.